src/helpers/bumpVersion.js

```diff
@@ -6,23 +9,40 @@
  * @returns {string}
  */
 module.exports = (releaseType, version) => {
-  let [major, minor, patch] = version.split('.')
+  let major, minor, patch
 
-  switch (releaseType) {
-    case 'major':
-      major = parseInt(major, 10) + 1
-      minor = 0
-      patch = 0
-      break
+  if (version) {
+    [major, minor, patch] = version.split('.')
 
-    case 'minor':
-      minor = parseInt(minor, 10) + 1
-      patch = 0
-      break
+    switch (releaseType) {
+      case 'major':
+        major = parseInt(major, 10) + 1
+        minor = 0
+        patch = 0
+        break
+
+      case 'minor':
+        minor = parseInt(minor, 10) + 1
+        patch = 0
+        break
 
-    default:
-      patch = parseInt(patch, 10) + 1
+      default:
+        patch = parseInt(patch, 10) + 1
+    }
+  } else {
+    let version = semverValid(core.getInput('fallback-version'))
+
+    if (version) {
+      [major, minor, patch] = version.split('.')
+    } else {
+      // default
+      major = 0
+      minor = 1
+      patch = 0
+    }
   }
 
+  core.info(`The version could not be detected, the new version is '${major}.${minor}.${patch}'.`)
```

Comment:
Should this not be inside the `else`? As it's logged now every time, see my suggestion

```suggestion
```